plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -79,7 +79,7 @@ public void setUp()
 
     private CachingJdbcClient createCachingJdbcClient(Duration cacheTtl, boolean cacheMissing)
     {
-        return new CachingJdbcClient(database.getJdbcClient(), Set.of(getTestSessionPropertiesProvider()), cacheTtl, cacheMissing);
+        return new CachingJdbcClient(database.getJdbcClient(), Set.of(() -> PROPERTY_METADATA), cacheTtl, cacheMissing);
```

Comment:
Please extract private constant for ` Set.of(() -> PROPERTY_METADATA)`, PROPERTY_METADATA_PROVIDERS?

Reply:
Done